tests/x25519_tests.rs

```diff
@@ -1,4 +1,3 @@
-
```

Comment:
All of the changes to this file seem unrelated and should probably go in a separate rustfmt PR

Reply:
will revert them then

Reply:
@tarcieri done